theano/gof/opt.py

```diff
@@ -1953,12 +2047,60 @@ def print_profile(stream, prof, level=0):
         print(blanc, "  init io_toposort", io_t, file=stream)
         print(blanc, "  loop time", loop_t, file=stream)
         print(blanc, "  callback_time", callback_time, file=stream)
+        if isinstance(lopt, LocalOptGroup):
+            lopt.print_profile(stream, (lopt.time_opts,
+                                        lopt.time_nodes,
+                                        lopt.process_count,
+                                        lopt.applied_true,
+                                        lopt.node_created))
```

Comment:
Add the level parameter to that function call.
level=level+1

Otherwise, you loose the indentation.